<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchDialog</class>
 <widget class="QDialog" name="BatchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CompuTreeBatch</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBoxPlugins"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStart">
       <property name="text">
        <string>Démarrer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStop">
       <property name="text">
        <string>Arrêter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonReload">
       <property name="text">
        <string>Recharger</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="GStepManager" name="widgetStepManager" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GStepManager</class>
   <extends>QWidget</extends>
   <header>StepResultTreeView/gstepmanager.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
